.github/workflows/rust.yml

```diff
@@ -14,17 +14,12 @@ jobs:
     runs-on: ubuntu-18.04
     steps:
       - uses: actions/checkout@230611dbd0eb52da1e1f4f7bc8bb0c3a339fc8b7
-      - uses: actions-rs/toolchain@88dc2356392166efad76775c878094f4e83ff746
```

Comment:
@zephraph I think this change is correct -- we don't have to specify this action any more in this job, right?

Reply:
I thought this action was just to get access to *any* rust toolchain, and we needed:

- "nightly-2021-03-25" here, and
- "Whatever is in the rust-toolchain.toml" file below

But it seems like, based on the "checks" output, we actually happen to be downloading the version we need (specified in the rust-toolchain.toml) on the first call to `cargo --version`.

Checking my understanding: With this new change, are we just relying on whatever happens to be installed on Github's default ubuntu-18.04 image to bootstrap?

Reply:
I don't believe we're relying on whatever happens to be installed because we have a rust-toolchain file in the repo.

Reply:
To expans on that: [the Actions environment, at least for Ubuntu 18.04, includes Rust 1.57](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu1804-Readme.md#rust-tools).  I believe this is updated pretty regularly when a new Rust release comes out.  So this will likely already be the right thing when this runs.  But if it's not (e.g., if they've updated to 1.58 and we have not yet updated our rust-toolchain file), then the Rust that _is_ installed by default will honor the rust-toolchain file like it always does, installing the specified toolchain and running the rest of the build with it.

Reply:
Right, that's what I meant by "bootstrap" in my comment above. Sounds reasonable, if we expect that'll always exist in the Actions environment across our supported test systems.

Reply:
I think that's right.  It's documented to be present in the Actions environments for Ubuntu, Windows, and Mac.  I would imagine it would be a pretty big breaking change if it were removed.  If it were, we could add this action back.

We might also add the action back if we wanted to override rust-toolchain to do builds with nightly or beta or some other stable (e.g., a MSRV).